__tests__/app.test.js

```diff
@@ -97,6 +97,76 @@ describe('App Tests', () => {
                 })
             })
         })
+        describe('Added Feature: GET `/api/articles` test', () => {
+            test('topic query: filters the articles by topic value specified in the query.', () => {
+                return request(app)
+                .get('/api/articles?topic=mitch')
+                .then(({body: {articles}}) => {
+                    expect(articles.length).not.toBe(0);
+                    articles.forEach(article => {
+                        expect(article.topic).toBe('mitch');
+                    })
+                })
+            })
+            test('topic query: filters the articles by topic value specified in the query and is not case-sensitive.', () => {
+                return request(app)
+                .get('/api/articles?topic=MiTcH')
+                .then(({body: {articles}}) => {
+                    expect(articles.length).not.toBe(0);
+                    articles.forEach(article => {                        
+                        expect(article.topic).toMatch(/MiTcH/i);
+                    })
+                })
+            })
+            test('sort_by query: sorts the articles by any valid column(defaults to created_at)', () => {
+                return request(app)
+                .get('/api/articles?sort_by=title')
+                .then(({body: {articles}})=> {
+                    expect(articles.length).not.toBe(0);
+                    expect(articles).toBeSortedBy('title', {descending: true});
+                    expect(articles).toEqual(articles.sort((a,b) => b.title - a.title));                    
+                })
+            })
+            test('sort_by query: sorts the articles by any valid column paired with topic query', () => {
+                return request(app)
+                .get('/api/articles?topic=mitch&sort_by=title')
+                .then(({body: {articles}})=> {
+                    expect(articles.length).not.toBe(0);
+                    expect(articles).toBeSortedBy('title', {descending: true});
+                    
+                    articles.forEach(article => {
+                        expect(article.topic).toBe('mitch');
+                    })
+                })
+            })
+            test('order query: can be set to asc for ascending', () => {
+                return request(app)
+                .get('/api/articles?sort_by=article_id&order=asc')
+                .then(({body: {articles}})=> {
+                    expect(articles.length).not.toBe(0);
+                    expect(articles).toBeSortedBy('article_id', {ascending: true});
```

Comment:
Looking at the docs I think you might want this to be `('article_id', { descending: false});` rather than _ascending: true_